options.js

```diff
@@ -1,26 +1,97 @@
 ((browser) => {
-  document.querySelector('h1').textContent =
-    browser.i18n.getMessage('link_copy_style');
+  document.querySelector('h1').textContent = browser.i18n.getMessage('link_copy_style');
   document.querySelector('.rich').textContent = browser.i18n.getMessage('rich');
   document.querySelector('.raw').textContent = browser.i18n.getMessage('raw');
+  document.querySelector('.rich_plus_raw').textContent = browser.i18n.getMessage('rich_plus_raw');
+  document.querySelector('h4').textContent = browser.i18n.getMessage('link_text');
+  document.querySelector('.link_text_option_1').textContent = browser.i18n.getMessage('link_text_option_1');
+  document.querySelector('.link_text_option_2').textContent = browser.i18n.getMessage('link_text_option_2');
+  document.querySelector('.link_text_option_3').textContent = browser.i18n.getMessage('link_text_option_3');
+  document.querySelector('.link_text_option_4').textContent = browser.i18n.getMessage('link_text_option_4');
```

Comment:
Here and throughout: please use `-` as the separator for class names per the [Google CSS style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters). Thanks!

Reply:
Done in af62d525855da94d9bbc8b0bb3a751c290660d4d